src/scripts/leaves.ts

```diff
@@ -15,7 +15,7 @@ class Leaves {
 
   constructor(public constantOpacity: boolean) {
     this.loadLeaves();
-    this.renderer = PIXI.autoDetectRenderer(800, 600, {backgroundColor : 0xfdf3e1, antialias: false});
+    this.renderer = PIXI.autoDetectRenderer(800, 600, {backgroundColor : 0x0000ff, antialias: false});
```

Comment:
No, just testing. PR doesn't mean it's final. Only snowflake sprites are relevant for now.